crates/proof-of-sql/src/base/database/owned_table_test_accessor_test.rs

```diff
@@ -130,7 +133,16 @@ fn we_can_access_the_commitments_of_table_columns() {
     let column = ColumnRef::new(table_ref_1, "b".parse().unwrap(), ColumnType::BigInt);
     assert_eq!(
         accessor.get_commitment(column),
-        compute_commitment_for_testing(&[4, 5, 6], 0_usize)
+        RistrettoPoint::compute_commitments(
+            &[CommittableColumn::Scalar(slice_cast::<
+                Curve25519Scalar,
+                [u64; 4],
+            >(&slice_cast(&[
+                4, 5, 6
+            ])))],
```

Comment:
You should be able to leverage the conversions we have in place for `CommittableColumn` to make it much more readable. See here: https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/e40362a2865fb9c95ea5bdfb9bbc48471500cb33/crates/proof-of-sql/src/base/commitment/committable_column.rs#L152-L188.

Something like this should work
```suggestion
            &[(&[4, 5, 6]).into()],
```